File: app/src/main/java/com/originb/inkwisenote/modules/noterelation/data/DocumentNode.java
package com.originb.inkwisenote.modules.noterelation.data;

import java.util.HashMap;
import java.util.Map;

public class DocumentNode {
    Long documentId;
    Map<String, Integer> termFrequencies; // Term frequencies within this document

    public DocumentNode(Long documentId) {
        this.documentId = documentId;
        this.termFrequencies = new HashMap<>();
    }
}

File: app/src/main/java/com/originb/inkwisenote2/modules/noterelation/data/DocumentNode.kt
package com.originb.inkwisenote2.modules.noterelation.data

class DocumentNode(var documentId: Long?) {
    var termFrequencies: MutableMap<String?, Int?>? // Term frequencies within this document

    init {
        this.termFrequencies = HashMap<String?, Int?>()
    }
}

